setup.py

```diff
@@ -59,6 +60,10 @@ def set_builtin(name, value):
                         ),
               ]
 
+install_requires = ['numpy', 'freetype-py', 'hsluv', 'kiwisolver', 'packaging']
+if sys.version_info <= (3, 8):
+    install_requires.append("importlib-resources")
```

Comment:
Apparently this was flawed. Let's try:

```suggestion
install_requires = ['numpy', 'freetype-py', 'hsluv', 'kiwisolver', 'packaging']
if sys.version_info < (3, 9):
    install_requires.append("importlib-resources")
```